pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/model/config/DropFieldsConfig.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.pulsar.functions.transforms.model.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import lombok.Getter;
+
+@Getter
+public class DropFieldsConfig extends StepConfig {
+  @JsonProperty(required = true)
+  private List<String> fields;
+
+  @JsonProperty(required = false)
+  private String part;
```

Comment:
Annotating with @JsonProperty (althought required = false is the defaul) avoid this error
```
[ERROR] Medium: Unwritten field: com.datastax.oss.pulsar.functions.transforms.model.config.DropFieldsConfig.part [com.datastax.oss.pulsar.functions.transforms.model.config.DropFieldsConfig] At DropFieldsConfig.java:[line 27] UWF_UNWRITTEN_FIELD
```

Another we to do it is to add Setter - although jackson deserialization doesn't need it.

Reply:
That's a spotbug error. Maybe it can deactivated for the Jackson files ?
Or annotate the class with `@Data`. It will make it mutable but is it a big deal ?

Reply:
Or annotate the class with
```java
@Jacksonized
@Builder
@Value
```
to keep it immutable. Jackson will use the builder (https://projectlombok.org/features/experimental/Jacksonized)